nibabel/minc1.py

```diff
@@ -269,6 +269,7 @@ class MincHeader(Header):
     """
     # We don't use the data layout - this just in case we do later
     data_layout = 'C'
+    sizeof_hdr = 4
```

Comment:
We previously had a `sniff_size` property, but `sniff_size` was equal to `sizeof_hdr` in all cases where they were both defined.

I see three options:
* change `sizeof_hdr` in all cases to be the size of the header (if not 4 above, then increase it).
* read `sniff_size` if it exists, otherwise, look for a `sizeof_hdr`, otherwise don't do the sniff check.
* revert back to the `sniff_size`, stop reading `sizeof_hdr` at all.

Any preferred way to go?